arboretum/kubernetes/README.md

```diff
@@ -17,7 +17,70 @@ to include the fetchers and checks from this library in your downstream project.
 
 ## Fetchers
 
-Fetchers coming soon...
+### Cluster Resource
+
+* Class: [ClusterResourceFetcher][fetch-cluster-resource]
+* Purpose: Write the resources of **stand-alone** Kubernetes clusters to the
+evidence locker.  **NOTE:** Do not use this fetcher for managed clusters.
+Instead use the [IBM Cloud cluster list fetcher][ibm-cloud-cluster-list-fetcher].
+* Behavior: Retrieve stand-alone Kubernetes cluster resource data for the provided
+list of clusters.  TTL is set to 1 day.
+* Configuration elements:
+  * `org.kubernetes.cluster_resources.clusters`
+    * Required
+    * List of dictionaries:
+      * `account name`: An arbitrary IBM Cloud account name as a string
+      * `server` A URL as a string to a Kubernetes stand-alone cluster
+  * `org.kubernetes.cluster_resources.types`
+    * Optional
+    * List of resource types as strings
+      * NOTE: For core group API resources, the resource name must be in
+      _plural form_.  See example below.
+      * NOTE: For other "custom" API resources, the resource name must be in the
+      following format: `"APIGROUP/VERSION/NAME"`.  See example below.
+    * Use if looking to override the default of `["nodes", "pods", "configmaps"]`.
+    Otherwise do not include.
+* Expected configuration:
+
+  ```json
+  {
+    "org": {
+      "kubernetes": {
+        "cluster_resources": {
+          "clusters": [
+            {
+              "account": "myaccount1",
+              "server": "https://myserver1:30000"
+            }
+          ],
+          "types": ["nodes", "pods", "apigroup.example.com/v1/mycustom"]
+        }
+      }
+    }
+  }
+   ```
+
+
+* Required credentials:
+  * Kubernetes API server token with read/view permissions are needed for this fetcher to successfully retrieve the evidence.
+    * `XXX_token`: token string for account `XXX`.
+    * Example credential file entry:
+
+      ```ini
+      [kubernetes]
+      myaccount1_token=your-kubernetes-api-token-1
+      myaccount2_token=your-kubernetes-api-token-2
+      ```
+
+    * Expected Travis environment variable settings to generate credentials:
+      * `KUBERNETES_MYACCOUNT1_TOKEN`
+      * `KUBERNETES_MYACCOUNT2_TOKEN`
```

Comment:
I would explain how you get the token here. Is the token really permanent? I thought it would expire eventually.

Reply:
I added descriptions including persistency of token.
https://github.com/ComplianceAsCode/auditree-arboretum/pull/32/files#diff-ab8d1d49d0939d2f9104b162f9cf493635b8dc57fff06c0e7f5dcdf8883c6f2bR66